gnosis/safe-react
# feat: Save network to sessionStorage

## What it solves
Resolves #2921

## How this PR fixes it
The current network is also saved in the `sessionStorage`.

## How to test it
Open the `/welcome` route, change the network and refresh. The network should remain the same.
